### DISCUSSION TITLE
Circuit Box connection name fields too short for localization

### DISCUSSION THREAD

**Discussion by Jade-Harleyy** [FakeFishGames/Barotrauma#14013] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

![image](https://github.com/FakeFishGames/Barotrauma/assets/67431770/c68eca62-d3f1-4d50-8975-0fe76538f538)

### Reproduction steps

1. Open the Submarine Editor.
2. Place a Circuit Box and open it.
3. Try to rename any connection to a long string.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.5.0.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/aa9d2461a9218b3fa4ac7a002652d0038ad441bc
